language/move-prover/tests/testsuite.rs

```diff
@@ -127,6 +136,7 @@ fn cvc4_deny_listed(path: &Path) -> bool {
         path_str == "../diem-framework/modules/DesignatedDealer.move" ||
         path_str == "tests/sources/functional/marketcap.move" ||
         path_str == "tests/sources/functional/invariants.move" ||
+        path_str == "tests/sources/functional/invariants_resources.move" ||
```

Comment:
Actually, `transaction_scripts/burn.move` also has the same issue to me. Could we also add it to the "cvc4_deny_list"?

```
...
failures:
    functional::functional/invariants_resources.move
    fx::transaction_scripts/burn.move
test result: FAILED. 140 passed; 2 failed; 0 filtered out
```

Reply:
Done. Moving forward, the cvc4 tests are anyway only run on explicit user request. We may consider adding them back to regular test runs, but than we really need a whitelist not a blacklist because no one adding new Diem scripts modules or test cases would expect cvc4 in the play.